operations/companies.md

```diff
@@ -89,7 +89,8 @@ Returns all company profiles of the enterprise, possibly filtered by identifiers
             "Name": "Booking.com",
             "IsActive": true,
             "TaxIdentifier": "",
-            "BillingCode": ""
+            "BillingCode": "",
+            "InvoicingEmail": null
```

Comment:
It's missing for the first company in the result collection. 

Not really related to this PR, but there is plenty of fields missing in the response of those that are not deprecated, and deprecated fields like 'Address' is included in the documentation. We are omitting the deprecated fields from the gitbook, so this was probably long forgotten... That's the issue with the manually maintained documentation 😢

Reply:
Hopefully we'll get to automate this, but even though it's not exactly related to this one, let me fix that real quick